internal/rbd/rbd_util_test.go

```diff
@@ -283,3 +284,44 @@ func TestStrategicActionOnLogFile(t *testing.T) {
 		})
 	}
 }
+
+func TestIsKrbdFeatureSupported(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name        string
+		featureName string
+		isSupported bool
+		err         error
+	}{
+		{
+			name:        "supported feature",
+			featureName: "layering",
+			isSupported: true,
+			err:         nil,
+		},
+		{
+			name:        "not supported feature",
+			featureName: "journaling",
+			isSupported: false,
+			err:         nil,
+		},
+		{
+			name:        "unrecognized feature",
+			featureName: "xyz",
+			isSupported: false,
+			err:         librbd.ErrNotFound,
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
```

Comment:
Please do not  overload the `tt` variable, it is very ugly and I hope other tests get cleaned up too. Use `tc := tt` and use `tc` afterwards, or something like that.